conceptual.md

```diff
@@ -0,0 +1,42 @@
+### Conceptual Exercise
+
+Answer the following questions below:
+
+- What is a JWT?
+
+   - Stands for JSON Web Token - a digitally signed token that represents json data and can be used to send the data securely.
+
+- What is the signature portion of the JWT?  What does it do?
+    - The signature is the third and last part of the JWT and represets a signature of the header, payload and a secret key
+
+- If a JWT is intercepted, can the attacker see what's inside the payload?
+    - If the intercepted JWT includes a valid signature then yes.
+
+- How can you implement authentication with a JWT?  Describe how it works at a high level.
+    - You can use JWT to sign and/or verify a token that represents a given user, and even include that token in future requests to signify that that user is currently logged in.
+
+- Compare and contrast unit, integration and end-to-end tests.
+    - UNIT: involves testing single functions to make sure they perform correctly by themselves
+    - INTEGRATION: using network responses to see if functions you have written work correctly together
```

Comment:
Doesn't necessarily have to include network responses but typically does. Integration tests typically test the integration of 3rd-party libraries into your code.